inc/lift/RequestHandle.h

```diff
@@ -183,6 +194,23 @@ class RequestHandle
      */
     auto GetUserData() -> void*;
 
+    /**
+     * Set to true if the maximum number of bytes has been written.
+     * @param max_bytes_written     Boolean indicating if the max number of bytes has been written.
+     */
+    auto SetCompletedWritingMaxBytes(bool max_bytes_written) -> void;
```

Comment:
is this an internal function?

Reply:
Changed to private function (same for getCompletedWritingMaxBytes)